predicators/nsrt_learning/strips_learning/oracle_clustering_learner.py

```diff
@@ -0,0 +1,86 @@
+"""Oracle for STRIPS learning."""
+
+from typing import List, Set, Tuple
+
+from predicators.envs import get_or_create_env
+from predicators.ground_truth_models import get_gt_nsrts, get_gt_options
+from predicators.nsrt_learning.strips_learning import BaseSTRIPSLearner
+from predicators.settings import CFG
+from predicators.structs import NSRT, PNAD, Datastore, DummyOption, \
+    LiftedAtom, ParameterizedOption, Variable
+
+
+class OracleSTRIPSLearner(BaseSTRIPSLearner):
+    """Base class for a STRIPS learner that uses oracle operators but re-learns
```

Comment:
can you explain here what "clustering" means in the context of this class? also, should we include Clustering in the name of the class to match the name of the file?

Reply:
actually we should definitely change it because `OracleSTRIPSLearner` is already defined elsewhere